app/src/app/routes/Sources/SourceMappings.tsx

```diff
@@ -37,6 +39,11 @@ const SourceMappings = (): JSX.Element => {
   const classes = useStyles();
   const history = useHistory();
   const { sourceId } = useParams<{ sourceId?: string }>();
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    dispatch(resourcesRemoved());
```

Comment:
why putting this here in this file ?

Reply:
When you where creating a new mapping, the mapping used in the fhirAttributePannel was prefilling everything because the mapping was still in the store. But what you told me to do after with `useApiResourcesRetrieveQuery` doesn't need this, so I can delete it, and the `resourceAdded` in the next comment.

Reply:
I deleted the comment you made in FhirAttributePannel.tsx file (my mistake) but I can remove the `resourceAdded` function since we don't need the store anymore.